front/lib/resources/public_ids.ts

```diff
@@ -0,0 +1,86 @@
+import type { ModelId, Result } from "@dust-tt/types";
```

Comment:
Let's not introduce a new terminology this is just a kind of sId no?

Reply:
I'd push for a new terminology here as this concept is different than the existing `sId` one. Having one work/terminology per concept is pretty efficient IMHO.